app/components/home-product-banners/home-product-banners.tsx

```diff
@@ -0,0 +1,104 @@
+import { Text, ButtonLink } from '~/components'
+import type { HomeProductBannersProps } from './types'
+
+const HomeProductBanners = ({ products }: HomeProductBannersProps) => {
+  const { primary, background, sideBySide } = products
+
+  return (
+    <div className="px-6 sm:px-10">
+      <div className="flex flex-col gap-6 max-w-6xl mx-auto sm:gap-8 lg:gap-12">
+        { primary && (
+          <div className="bg-orange rounded-lg px-6 py-14 lg:py-32 lg:px-24 lg:flex lg:justify-end lg:items-center lg:relative">
+            { primary.contents.home_image && (
+              <div className="mb-8 sm:mb-16 lg:absolute lg:bottom-0 lg:mb-0 lg:left-32">
+                <img
+                  src={primary.contents.home_image[0]}
+                  alt=''
+                  className="h-52 mx-auto lg:h-128"
+                />
+              </div>
+            )}
+
+            <div>
+              <Text variant="heading-1" className="!text-4.5xl sm:!text-6xl text-white text-center mb-6 w-40 mx-auto sm:w-80 lg:text-left" as="h2">
+                { primary.name }
+              </Text>
+
+              { primary.contents.home_description && (
+                <Text variant="body" className="font-medium opacity-75 text-white text-center mb-6 sm:w-80 sm:mx-auto sm:mb-10 lg:text-left">
+                  { primary.contents.home_description }
+                </Text>
+              )}
+
+              <ButtonLink to={`/products/${primary.slug}`} variant="secondary" className="w-44 mx-auto text-center !bg-black hover:!bg-white !text-white hover:!text-black lg:mx-0">
+                See Product
+              </ButtonLink>
+            </div>
+          </div>
+        )}
```

Comment:
Maybe, we can fragment HomeProductBanners, into components for each condition. No?

This could be one of the subdivided components

Reply:
good call, done!